interface/templates/interface/generics/navbar.html

```diff
@@ -1,24 +1,34 @@
 <nav class="navbar navbar-expand-xl navbar-light bg-light">
   <a class="navbar-brand">VMCK</a>
+
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
+
   <div class="collapse navbar-collapse" id="navbarNav">
     <ul class="navbar-nav">
+      {% if current_user.is_authenticated %}
```

Comment:
Please add some indentation for the stuff inside `if` and `else` so it's easier to figure out which is which.